core/src/test/java/org/testcontainers/dockerclient/ImagePullTest.java

```diff
@@ -0,0 +1,58 @@
+package org.testcontainers.dockerclient;
+
+import org.junit.Test;
+import org.testcontainers.containers.GenericContainer;
+
+public class ImagePullTest {
```

Comment:
this test seems to miss Parameterized runner a lot ;)

Reply:
Refactored :D